src/main/webapp/app/admin/health/health-modal.component.ts

```diff
@@ -12,7 +12,7 @@ export class HealthModalComponent {
 
     constructor(public activeModal: NgbActiveModal) {}
 
-    readableValue(value: number): string {
+    readableValue(value: any): string {
```

Comment:
is this change really needed? we would like to avoid any as type

Reply:
I cannot really assure that `value` is a number here. `readableValue()` is used only once with a very nested attribute called `health.value.details[].value` where details is of type any. I could declare details as a { key, value } object but as the test instantiate this value with only a string, i decided i will use this safer method and declare this parameter as any instead.